Makefile

```diff
@@ -494,3 +494,9 @@ endif
 
 $(DIST_DIR)/CHANGELOG.md: $(ROOT_DIR)/cliff.toml bin/git-cliff | $(DIST_DIR)
 	$(GITCLIFF_BIN) --config $(ROOT_DIR)/cliff.toml --output $@ $(GITCLIFF_OPTS)
+
+# helper to create a comma-separated list from a space-separated list
+null  :=
+space := $(null) #
+comma := ,
+subst-space-with-comma = $(subst $(space),$(comma),$(strip $(1)))
```

Comment:
I'd move this part to `makefiles.d/10-helpers.mk` to avoid polluting the `Makefile`.

Reply:
Thanks, I didn't notice we have this file 👍